var/spack/repos/builtin/packages/singularity/package.py

```diff
@@ -24,8 +24,11 @@ class Singularity(MakefilePackage):
     url      = "https://github.com/sylabs/singularity/releases/download/v3.1.1/singularity-3.1.1.tar.gz"
     git      = "https://github.com/sylabs/singularity.git"
 
+    maintainers = ['ArangoGutierrez','alalazo']
+    
     version('develop', branch='master')
```

Comment:
By the way, now that #1983 has been merged, you can actually name this version `master` if you want to be consistent.

Reply:
feel free to do the github suggestion :)

Reply:
I am on UI mode only this afternoon so hard to go to my editor

Reply:
```suggestion
    version('master', branch='master')
```